float64 rotation_angle
string input_pcd_file
string output_pcd_file
---
bool success
